src/txs/Tx.tsx

```diff
@@ -171,14 +177,18 @@ function Tx<TxValues>(props: Props<TxValues>) {
     (denom: CoinDenom) => {
       const gasPrice = chain?.startsWith("carbon-")
         ? carbonFees?.prices[denom]
+        : chain?.startsWith("osmosis")
+        ? (osmosisGas?.base_fee || 0.0025) * 10
```

Comment:
And maybe move this logic inside the hook itself, with `0.0025` used as default value inside the `catch` when the LCD is down
```
(osmosisGas?.base_fee || 0.0025) * 10
```

Reply:
moved try/catch into hook, however, hook returns UseQueryResult which is empty until promise resolves. added  default while waiting for data.